.github/workflows/deploy_staging.yml

```diff
@@ -0,0 +1,48 @@
+name: Deploy staging app to Firebase App Distribution
+on:
+  push:
+    branches:
+      - develop
```

Comment:
Can you please help to skip running the `test` workflow on `develop, release/ and main` branches?

https://github.com/nimblehq/kmm-templates/blob/9bf253c53a000a99f6ef59b15251a7fb532d4af7/.github/workflows/test.yml#L2C4-L4

Reply:
@luongvo Did you mean you would like to update the `test` script along with this PR?

Reply:
You got the answer for this one 🙏